### ISSUE TITLE
Issues with places name which have spaces inbetween ? 

### ISSUE THREAD

**Issue by kshitijrajsharma** [wipfli/nepali-map-labels#1]
<img width="1331" alt="image" src="https://github.com/wipfli/nepali-map-labels/assets/36752999/d2845bd4-0684-4c6a-a4fb-a2c31e7725c5">

First of all thank you for this  , It looks great , most of the places rendering issues are solved , I found one issue may be its because of the space inbetween in label . 

POI Details : https://www.openstreetmap.org/node/4382235692

**Comment by kshitijrajsharma** [comment]
Name seems fine here on the input : 
https://github.com/wipfli/nepali-map-labels/blob/a638ced498f5e3fc5469acb2a89042ae2796bf95/cities.geojson#L3092

**Comment by wipfli** [comment]
Hi @kshitijrajsharma. Thank you for the feedback. You are correct, in this map labels with spaces only show the first word. Sorry for this.

I have a new map with hopefully better text. You find it at https://github.com/wipfli/maplibre-precomposed-i18n/. There are also some instructions how to show all labels on the globe in Nepali.

**Comment by wipfli** [comment]
![image](https://github.com/wipfli/nepali-map-labels/assets/53421382/a9001a7b-cd9c-4e7c-814c-df8c8b75187e)

Now works with a simple URL:

https://wipfli.github.io/maplibre-precomposed-i18n/examples/global/#map=11.92/27.70187/85.25517&language=ne

**Comment by kshitijrajsharma** [comment]
Perfect @wipfli  Thanks for this , Just to confirm , Now it will work with any version of Maplibre with the style provided on the repo , right ?

**Comment by wipfli** [comment]
Yes. Also MapLibre Native and also any other renderer which can consume MapLibre styles such as maybe QGIS or other programs...
